src/main/java/duke/command/ByeCommand.java

```diff
@@ -0,0 +1,28 @@
+package duke.command;
+
+/**
+ * Represents a "bye" command that is entered by the user.
+ */
+public class ByeCommand extends Command {
+    private static final String EXIT_MESSAGE = "Goodbye. Hope that I have managed to scare all your problems away."
+            + "\nHave a great day! :)";
+
+
+    /**
+     * Constructs a <code>ByeCommand</code>.
```

Comment:
You can instead use `{@code ByeCommand}` - it will be more concise.

Reply:
Okay, thanks for the suggestion!